src/Components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,91 @@
+// Themes
+@import '../../constants/Themes/MainTheme';
+
+.tooltipContainer {
+  position: relative;
+  display: inline-block;
+  color: white;
+
+  .tooltipMsg {
+    background: $Color-TooltipBG;
+    border-radius: 5px;
+    border: solid 1px darkgray;
+    display: inline-block;
+    margin: 0;
+    padding: 4px 8px;
+    position: absolute;
+    text-align: left;
+    visibility: hidden;
+    width: max-content;
+
+    &.visible {
+      visibility: visible
+      ;
+    }
+
+    &::after {
+      content: " ";
+      border-width: 5px;
+      border-style: solid;
+      position: absolute;
+    }
+
+    &.top {
+      bottom: 130%;
+      left: 10%;
+
+      &::after {
+        border-color: black transparent transparent transparent;
+        left: 10px;
+        margin-left: -5px;
+        top: 100%;
+      }
+    }
+
+    &.bottom {
+      left: 10%;
+      top: 130%;
+
+      &::after {
+        bottom: 100%;
+        border-color: transparent transparent black transparent;
+        left: 10px;
+        margin-left: -5px;
+      }
+    }
+
+    &.left {
+      right: 100%;
+
+      &::after {
+        border-color: transparent transparent transparent black;
+        margin-top: -5px;
+        left: 100%;
+        top: 10px;
+      }
+    }
+
+    &.right {
+      left: 105%;
+
+      &::after {
+        border-color: transparent black transparent transparent;
+        margin-top: -5px;
+        right: 100%;
+        top: 10px;
+      }
+    }
+
+    .tooltipText {
+      margin: 0;
+      overflow-y: auto;
+      padding: 0;
+    }
+  }
+
+  &:hover {
+    .tooltipMsg {
+      visibility: visible;
+    }
+  }
+}
```

Comment:
Please sort styles in alphabetical order

Reply:
Done.